getting-started/kubernetes/trying-ebpf.md

```diff
@@ -0,0 +1,190 @@
+---
+title: Try out the eBPF tech preview
+description: Try out the eBPF dataplane tech preview.
+---
+
+### Big picture
+
+Install {{site.prodname}} and enable the tech preview of the new eBPF dataplane.
+
+> **Warning!** eBPF mode is a tech preview and should not be used in production clusters. It does not support all the features of {{site.prodname}} and it is missing some security features (for example anti-spoofing protection).
+{: .alert .alert-danger }
+
+### Value
+
+The new eBPF dataplane mode has several advantages over iptables mode:
+
+* It scales to higher throughput.
+* It uses less CPU per GBit.
+* It has native support for Kubernetes services (without needing kube-proxy) that:
+
+  * Reduces first packet latency for packets to services.
+  * Preserves external client source IP addresses all the way to the pod.
+  * Supports DSR (Direct Server Return) for more efficient service routing.
+  * Uses less CPU than kube-proxy to keep the dataplane in sync.
+  
+Trying out the tech preview will give you a taste of these benefits and an opportunity to give feedback to the {{site.prodname}} team. 
+
+### Features
+
+This how-to guide uses the following {{site.prodname}} features:
+
+- **calico/node**
+- **eBPF dataplane**
+
+### Concepts
+
+#### eBPF
+
+eBPF is a technology that allows you to write mini programs that can be attached to various low-level hooks in the Linux kernel, for a wide variety of uses including networking, security, and tracing. You’ll see a lot of non-networking projects leveraging eBPF, but for {{site.prodname}} our focus is on networking, and in particular, pushing the networking capabilities of the latest Linux kernel’s to the limit.
+
+#### eBPF mode manifest
+
+To make it easier to try out the tech preview, we have created a custom Kubernetes manifest (.yaml file) with eBPF mode pre-enabled.
+
+### Before you begin...
+
+In the tech preview release, eBPF mode has the following pre-requisites:
+
+- A v5.3+ Linux kernel, for the tech preview release, we have limited our testing to Ubuntu 19.10 so we strongly recommend that you start with that too.
+- An underlying network that doesn't require Calico to use an overlay.  The instructions below guide you through setting up a cluster in a single AWS subnet; an alternative would be to set up your cluster on-prem with a routed network topology.  Using the IPIP or VXLAN overlay modes add additional overhead making it harder to see the benefits of eBPF mode. 
+- The network must be configured to allow {{site.prodname}} packets between Calico hosts.  VXLAN is used to forward packets from outside the cluster that are destined for NodePorts and ClusterIPs.
```

Comment:
Omit (or reword?) VXLAN sentence. It's hard to explain without a diagram and not sure it's worth a diagram here given the likelihood of a network blocking VXLAN packets host-to-host?